RadixWallet/Features/TransactionReviewFeature/TransactionReview+Sections.swift

```diff
@@ -156,10 +159,64 @@ extension TransactionReview {
 			)
 
 		case let .validatorStake(validatorAddresses: validatorAddresses, validatorStakes: validatorStakes):
```

Comment:
those two big code hunks below are near identical? possible to share code?

Reply:
Not really, without restructuring the whole switch, because they have different associated types in the cases, they initialise the result in different ways, and both deposits and withdrawals are different. Generally speaking we have restructured it to avoid code duplication quite well, but most of the arguments to the extraction methods tend to be the same. Haven't found a good way to make use of that fact!